package.json

```diff
@@ -1,73 +1,88 @@
 {
   "name": "platform",
   "private": true,
-  "workspaces": [
-    "packages/*"
-  ],
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ],
+    "nohoist": [
+      "example-app/@angular-redux/store",
+      "example-app/@angular-redux/form",
+      "example-app/@angular-redux/router",
+      "example-app/ng-packagr"
+    ]
+  },
   "scripts": {
     "build": "npm-run-all build:*",
-    "build:store": "ng-packagr -p packages/store",
-    "build:form": "ng-packagr -p packages/form",
-    "build:router": "ng-packagr -p packages/router",
-    "release:canary": "lerna publish --canary --npm-tag next --npm-client npm",
+    "build:store": "yarn workspace @angular-redux/store build",
+    "build:form": "yarn workspace @angular-redux/form build",
+    "build:router": "yarn workspace @angular-redux/router build",
+    "release:canary": "lerna publish --canary --contents dist --dist-tag next",
```

Comment:
Canary builds were failing with yarn. See https://circleci.com/gh/angular-redux/platform/356. `--npm-client npm` needs to be kept